### DISCUSSION TITLE
Actividad 0 - Pavoni Nicolás

### DISCUSSION THREAD

**Discussion by NicoPavoni** [frlp-utn-ingsoft/catedra#24] [Show and tell]
https://github.com/NicoPavoni/IngSoftware

Algo a mencionar es que sin darme cuenta manejé mal el tema de la Nomenclatura con la numeración de las partes. No me di cuenta e hice todo Parte 1.X

**Comment by RodrigoJacznik** [comment]
Buenas.
Los commits y los mensajes podrían estar un poco mejor (más atómicos, mensajes mas claros).
Corri la app y me tiro un error. Fíjate que te falto cerrar el for del archivo mysite/polls/templates/polls/index.html.
Por favor fixealo para aprobar el trabajo.

Saludos.

**Reply by NicoPavoni** [reply]
Buenas, ahí subí el fix, probé creando una pregunta nueva para la encuesta desde la sección de admin y funcionó tanto la creación como la parte de la votación.
Si queda algo mas me avisas.
Gracias !

**Reply by RodrigoJacznik** [reply]
Buenas, ahora si.
El tp esta aprobado.
Saludos

**Reply by RodrigoJacznik** [reply]
Muy bien haciendo hasta la parte 7 👏 👏
